web_src/js/webcomponents/overflow-menu.ts

```diff
@@ -218,6 +224,12 @@ window.customElements.define('overflow-menu', class extends HTMLElement {
     }
   }
 
+  attributeChangedCallback() {
+    if (!this.button || !this.tippyContent) return;
+    const containActiveInTippy = this.tippyContent.querySelector('.item.active');
+    toggleClass(this.button, 'active', Boolean(containActiveInTippy));
```

Comment:
This logic should be in `updateItems`?

For example: when resizing, an active goes into the popup?